packages/common/src/llm/schemas.ts

```diff
@@ -44,21 +44,41 @@ const MessageSchema = z.object({
 export type Message = z.infer<typeof MessageSchema>
 
 export const GenerateContentInputSchema = z.object({
-  model: z.string(),
-  systemPrompt: z.string().optional(),
-  messages: z.array(MessageSchema),
+  model: z.string().describe('Model identifier to be used for content generation'),
```

Comment:
🤩  Thanks for improving the schema to better work with Zui, I didn't know these features were now available.

Reply:
Yup it's getting better and better daily! we're getting close to zui 1.0 :)